app/helpers/languages_helper.rb

```diff
@@ -194,7 +194,10 @@ module LanguagesHelper
     jbo: ['Lojban', 'la .lojban.'].freeze,
     kab: ['Kabyle', 'Taqbaylit'].freeze,
     ldn: ['Láadan', 'Láadan'].freeze,
+    ltg: ['Latgalian', 'latgalīšu'].freeze,
     lfn: ['Lingua Franca Nova', 'lingua franca nova'].freeze,
+    prg: ['Prussian', 'prūsiskai'].freeze,
```

Comment:
Wikipedia translates the language name as "Old Prussian", would it be better here?

Also there does not seem to be a lot of native speakers for this language. Is there a real interest for it to be added to Mastodon?